src/views/HowToPlay/Layout/Instructions-Layout.tsx

```diff
@@ -1,9 +1,10 @@
 import { KeepReading, BackButton } from '@ui/components';
 import { Outlet } from 'react-router';
+import StyledInstructionsLayout from './StyledInstructionsLayout';
 
 export const Home = () => {
```

Comment:
To avoid confusing ourselves, let's name this component the same as the filename. And remove the dash ("-") character from the filename.

```suggestion
export const InstructionsLayout = () => {
```